x-pack/metricbeat/module/gcp/billing/_meta/docs.asciidoc

```diff
@@ -1,18 +1,28 @@
 `billing` metricset is designed for collecting billing metrics from Google Cloud
 BigQuery daily cost detail table. BigQuery is a fully-managed, serverless data
 warehouse.
-Cloud Billing export to BigQuery enables you to export detailed Google Cloud
+Cloud Billing export to BigQuery enables you to export standard and detailed Google Cloud
 billing data (such as usage, cost estimates, and pricing data) automatically
 throughout the day to a BigQuery dataset that you specify. Then you can access
 your Cloud Billing data from BigQuery for detailed analysis using Metricbeat.
 Please see https://cloud.google.com/billing/docs/how-to/export-data-bigquery[export
 cloud billing data to BigQuery] for more details on how to export billing data.
 
-In BigQuery dataset, detailed Google Cloud daily cost data is loaded into a data
-table named `gcp_billing_export_v1_<BILLING_ACCOUNT_ID>`. There is a defined
-schema for Google Cloud daily cost data that is exported to BigQuery. Please see
-https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables#data-schema[
-daily cost detail data schema] for more details.
+In BigQuery, Google Cloud daily cost data is categorized into two formats:
+standard and detailed. Each format is stored within a designated dataset and
+follows a structured schema for precise cost analysis. For a comprehensive
+understanding of these formats, consult the https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/standard-usage#standard-usage-cost-data-schema[
+standard] and https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage#detailed-usage-cost-data-schema[
+detailed] data schema documentation.
+
+For regular cost monitoring and analysis, standard tables utilize the
+`gcp_billing_export_v1_<BILLING_ACCOUNT_ID>` format, presenting a comprehensive
+view of Google Cloud expenses.
+
+Alternatively, detailed tables, identifiable by the `gcp_billing_export_resource_v1_<BILLING_ACCOUNT_ID>`
+format, offer richer data. They include additional fields, such as `effective_price`,
+enabling a more granular view of expenses.
+
 
```

Comment:
We should explicitly state, "To get X, you must set Y". 

So "To get regular cost monitoring and analysis, you must set table pattern to `gcp_billing_export_v1_<BILLING_ACCOUNT_ID>` format", if I understand correctly.